pyproject.toml

```diff
@@ -35,13 +34,78 @@ dependencies = [
 
 [project.optional-dependencies]
 test = [
-    "pyfakefs~=5.2",
+    "pyfakefs~=5.2"
 ]
+lint = ["ruff"]
 
 [project.urls]
 "Homepage" = "https://github.com/us-irs/cfdp-py"
 
+[tool.ruff]
+exclude = [
+    ".git",
+    "venv",
+    "docs"
+]
+line-length = 120
```

Comment:
I'm not a fierce proponent of the 88 width rule, but can we keep this at 100 for now?
I prefer this because the code remains readable on side-side views on a laptop as well.

Also, if we change this, black should probably be configured accordingly as well.

Reply:
Changed it to 100. 
One note to black. Why not use ``ruff format``? It's already there and the config is there as well.

Reply:
Habits :) I suppose I could also use `ruff format` .. Then only one tool needs to be configured.